addons/website_sale/controllers/main.py

```diff
@@ -546,8 +546,6 @@ def resequence_product_image(self, image_res_model, image_res_id, move):
 
```

Comment:
It could be worth to add a docstring to the method:
```py
def resequence_product_image(self, image_res_model, image_res_id, move):
    """ Move the product image in the given direction and update all images' sequence.

    :param str image_res_model: The model of the image. It can be 'product.template',
                                'product.product', or 'product.image'.
    :param str image_res_id: The record ID of the image to move.
    :param str move: The direction of the move. It can be 'first', 'left', 'right', or 'last'.
    :raises NotFound: If the user does not have the required permissions, if the model of the
                      image is not allowed, or if the move direction is not allowed.
    :raise ValidationError: If the product is not found.
    :raise ValidationError: If the image to move is not found in the product images.
    :return: None
    """
```

Reply:
Done. I also documented the validation error which is throw when a video is moved to the first position.